projects/plugins/jetpack/modules/masterbar/admin-menu/admin-menu.js

```diff
@@ -89,25 +89,60 @@
 				}
 			} );
 
-			makeAjaxRequest(
-				'GET',
-				ajaxurl + '?action=upsell_nudge_jitm&_ajax_nonce=' + jetpackAdminMenu.upsellNudgeJitm,
-				undefined,
-				null,
-				function ( xhr ) {
-					try {
-						if ( xhr.readyState === XMLHttpRequest.DONE ) {
-							if ( xhr.status === 200 && xhr.responseText ) {
-								adminMenu
-									.querySelector( '#toplevel_page_site_card' )
-									.insertAdjacentHTML( 'afterend', xhr.responseText );
+			const siteUrl = new URL( jetpackAdminMenu.siteUrlOption );
+			let siteHostname = null;
+
+			if ( 'hostname' in siteUrl ) {
+				siteHostname = siteUrl.hostname;
+			}
+
+			if ( jetpackAdminMenu.launchpadScreenOption === 'full' ) {
+				const LaunchpadNotice =
+					'<li class="wp-not-current-submenu menu-top menu-icon-generic toplevel_page_site-notices" id="toplevel_page_site-notices">' +
+					`	<a href="https://wordpress.com/setup/${ jetpackAdminMenu.siteIntentOption }/launchpad?siteSlug=${ siteHostname }" class="wp-not-current-submenu menu-top menu-icon-generic toplevel_page_site-notices">` +
+					'		<div class="wp-menu-arrow">' +
+					'			<div></div>' +
+					'		</div>' +
+					'		<div class="wp-menu-image dashicons-before dashicons-admin-generic" aria-hidden="true"><br></div>' +
+					'			<div class="wp-menu-name">' +
+					'				<div class="upsell_banner">' +
+					'					<div class="banner__info">' +
+					'						<div class="banner__title">' +
+					'							Keep setting up your site' +
+					'						</div>' +
+					'					</div>' +
+					'				<div class="banner__action">' +
+					'					<button type="button" class="button">Next Steps</button>' +
+					'				</div>' +
+					'			</div>' +
+					'		</div>' +
+					'	</a>' +
+					'</li>';
+
+				adminMenu
+					.querySelector( '#toplevel_page_site_card' )
+					.insertAdjacentHTML( 'afterend', LaunchpadNotice );
+			} else {
```

Comment:
Note: Everything below this line was originally present in the file. We just place it behind a conditional clause